model_compression_toolkit/target_platform_capabilities/schema/schema_compatability.py

```diff
@@ -33,19 +34,57 @@ def is_tpc_instance(tpc_obj_or_path: Any) -> bool:
     return type(tpc_obj_or_path) in all_tpc_types
 
 
-def _schema_v1_to_v2(tpc: schema_v1.TargetPlatformCapabilities) -> schema_v2.TargetPlatformCapabilities:  # pragma: no cover
+def get_schema_by_version(schema_version: int):
+    return {
+        1: schema_v1,
```

Comment:
maybe use strings for the keys, we don't know what kind of versioning there's going to be in the future so it's better to have option for more complicated structure